website/project/views/node.py

```diff
@@ -513,18 +515,13 @@ def project_reorder_components(node, **kwargs):
     logger.error('Got invalid node list in reorder components')
     raise HTTPError(http.BAD_REQUEST)
 
-
-##############################################################################
-
-
 @must_be_valid_project
 @must_be_contributor_or_public
 @must_not_be_retracted_registration
-@ember_flag_is_active(features.EMBER_PROJECT_ANALYTICS)
 def project_statistics(auth, node, **kwargs):
-    ret = _view_project(node, auth, primary=True)
-    ret['node']['keenio_read_key'] = node.keenio_read_key
-    return ret
+    if 'project' in request.url:
```

Comment:
Can you explain why this is necessary? Was this broken before?

Nit: this check can be made correct using 

```suggestion
    if request.path.startswith('/project/'):
```

Reply:
I was trying to preserve the old route. Locally, if you navigated to project/<project_id>/forks and the analytics waffle flag was True, it would proxy to the ember app and you'd get a NotFound in the ember app.  I didn't think the legacy route was in the ember app, so I modified the route here.

Edit: it works on prod though, i'm not sure how.